tests/retrieveBearerToken.test.js

```diff
@@ -0,0 +1,109 @@
+import { createMocks } from "node-mocks-http";
+import client from "next-auth/client";
+import { retrieve } from "../pages/api/id/[form]/bearer";
+import fetchMock from "jest-fetch-mock";
```

Comment:
Lets discuss Monday about these tests because I'm not exactly sure they're testing what needs to be tested